README.md

```diff
@@ -33,6 +47,37 @@ If you wish to get more information about any command within `rdme`, you can exe
 - `--key <string>`: The API key associated with your ReadMe project. You can obtain this from your dashboard, or alternatively if you log in with `rdme login`, we will save your API key to a local configuration file (`~/.config/configstore/rdme-production.json`), saving you the hassle of having to supply this argument on commands that have it.
 - `--version <string>`: Your project version.
 
+### GitHub Actions
+
+<!-- TODO: update this docs link if needed! -->
+
+> For a full GitHub Workflow file example and additional information on GitHub Actions usage, check out [our docs](https://docs.readme.com/docs/automatically-sync-api-specification-with-github).
+
+For usage in GitHub Actions, create [a new GitHub Workflow file](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) in the `.github/workflows` directory of your repository and add the following [steps](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idsteps) to your workflow:
+
+```yml
+- uses: actions/checkout@v2
+- uses: readmeio/rdme@XX
+  with:
+    rdme: [your command here]
+```
+
+The command syntax in GitHub Actions is functionally equivalent to the CLI. For example, take the following CLI command:
+
+```sh
+rdme openapi path/to/openapi.json --key=API_KEY --id=OPENAPI_ID
+```
+
+To execute this command via GitHub Actions, the step would look like this:
+
+```yml
+- uses: readmeio/rdme@XX
+  with:
+    rdme: openapi path/to/openapi.json --key=API_KEY --id=OPENAPI_ID
```

Comment:
In this example, could you show to use a GitHub Actions secret in place of the variables?

Reply:
I thought about it but I opted not to because I was worried it would make the illustration of the usage harder to follow. I was thinking of having a lengthier section on using GitHub secrets in [our dedicated docs page](https://docs.readme.com/docs/automatically-sync-api-specification-with-github). Thoughts?

Reply:
¯\_(ツ)_/¯ That sounds useful but also seems a bit beyond the scope of our docs page. You could probably keep it simple and just link to here: https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow

Reply:
Oh yeah to clarify, "lengthier" is doing a lot of work in that last comment lol... I was planning on linking to that GitHub doc page and just including another example that uses secrets!